go.mod

```diff
@@ -130,6 +130,7 @@ require (
 	github.com/minio/md5-simd v1.1.2 // indirect
 	github.com/minio/sha256-simd v1.0.0 // indirect
 	github.com/mitchellh/mapstructure v1.4.3 // indirect
+	github.com/moby/spdystream v0.2.0 // indirect
```

Comment:
Do we need to update https://github.com/minio/operator/blob/master/CREDITS with license of new entry here?